### PR TITLE
Complete the design of instructions page

### PR DESCRIPTION
**Pull request by naveenpanickal**
## Description
Create a new page to display safety instructions.
Story: https://quarantinehelp.atlassian.net/browse/QH-39
### Prototype
[Quarantined](https://xd.adobe.com/view/7c6474dd-3491-4dc1-b5f9-4c91301be22e-2937/screen/e8d99ae9-7c8f-4f8a-b9e7-60796c486a03/specs/)
instrproto1

[Volunteer](https://xd.adobe.com/view/7c6474dd-3491-4dc1-b5f9-4c91301be22e-2937/screen/7c200e05-8546-4a95-a32a-98d5cb9ff0f2/specs/)
instrproto2

#### Quarantined
instructionQ

#### Volunteer
instructionV

## Related Issue

- Ionic back button not getting displayed

### CODE REVIEW

**Comment by sebinbenjamin** [comment]
Great work !. Please do have a look at the following

1. Drop shadow for the safety tips box.
2. Line height of the text in the safety instructions in the box.
3. Color/font-weight of the instruction step explanation paragraph text.
4. Color/importance of the dark circle for the step numbers.

**Comment by naveenpanickal** [comment]
> Great work !. Please do have a look at the following
> 
> 1. Drop shadow for the safety tips box.
> 2. Line height of the text in the safety instructions in the box.
> 3. Color/font-weight of the instruction step explanation paragraph text.
> 4. Color/importance of the dark circle for the step numbers.

Done

File: src/app/app.component.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ export class AppComponent {
       }
     } else if (this.hasUserOnboarded === true && this.userType === undefined) {
       console.log('New onboarded user to register');
-      this.router.navigateByUrl('/select-user-type');
+      this.router.navigateByUrl('/instructions');
```
Comment (sebinbenjamin) [inline src/app/app.component.ts:80]: I think the redirection to instructions page is to be called from create-requests or accept request page, not set here?

**Review by sebinbenjamin — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 1 above)
Comment (naveenpanickal) [reply src/app/app.component.ts:80]: here it should be  this.router.navigateByUrl('/select-user-type');
I changed it just to see the instructions page.

**Review by sebinbenjamin — APPROVED** [review]

**Pull request merged**